builtin/logical/database/rotation.go

```diff
@@ -64,6 +64,29 @@ func (b *databaseBackend) populateQueue(ctx context.Context, s logical.Storage)
 			continue
 		}
 
+		// If an account's NextVaultRotation period is nil, it means that the
+		// role was created before we added the `NextVaultRotation` field. In this
+		// case, we need to calculate the next rotation time based on the
+		// LastVaultRotation and the RotationPeriod. However, if the role was
+		// created with skip_import_rotation set, we need to use the current time
+		// instead of LastVaultRotation because LastVaultRotation is 0
+		if role.StaticAccount.NextVaultRotation.IsZero() {
+			log.Debug("NextVaultRotation is zero", roleName)
```

Comment:
I wonder if this log message could be extended a bit, what does this actually mean for the operator.

Reply:
Yea I agree with you. Is `NextVaultRotation` is nil it means the static role was created on a version of vault where the `NextVaultRotation` property didn't exist yet (pre 1.15.0).